### DISCUSSION TITLE
newer builds missing "changes"

### DISCUSSION THREAD

**Discussion by honx4amiga** [86Box/86Box#1830] [General]
since this "update" most builds are missing change log. the first build showing changes seems to be build #3257.
#3258 missing changes again, 3259 shows changes. there are no changes shown in 3268, 3269, 3271, 3272, 3274.

and as far i can see there are absolutely NO changes at all available for all builds prior to build #3257.
in other words, build #3256 and older seem to have lost all their change logs.

screenshot showing latest build #3274 without changes:
![86box_build_nochange](https://user-images.githubusercontent.com/82286022/142374654-b836b764-56f8-4c1c-92c3-0f08fbbf63f7.PNG)

another screenshot showing changes tab missing builds:
![86box_changes](https://user-images.githubusercontent.com/82286022/142375276-39f33d82-0265-4ded-8cc8-4fad46f3bd83.PNG)

changes tab also only starts at #3257.

**Comment by honx4amiga** [comment]
btw. what exactly is that new "86Box-DevODR" branch recently introduced?

it shares exactly the same descripion with experimental 86Box-Dev branch:
"These are development builds with experimental features. If you're looking for something more stable, use the standard builds instead. See the 86Box documentation for more information."

**Reply by OBattler** [reply]
That branch is -Dev but with the old recompiler while regular -Dev has the new one. The -Dev and -DevODR branches are being phased out, anyway.

**Comment by richardg867** [comment]
The changelogs have been restored. They were missing before due to a Jenkins migration oversight.

The issue with some recent builds not having any changes was also hopefully fixed for future builds.

**Comment by honx4amiga** [comment]
what changes were made in builds #3256, #3258, #3268, #3269, #3271, #3272 and #3274 which still remain missing?

another question regarding 86box for windows: is still x86 recommended as "normal" version to download over x64?
times were easier when only one version showed up for download instead of all build artefacts. now it's confusing. :D

**Reply by Cacodemon345** [reply]
x86 remains recommended to download over x64 as it is less buggy and has more performance over the x64 version.

**Comment by honx4amiga** [comment]
what changes were made in builds #3256, #3258, #3268, #3269, #3271, #3272 and #3274 which still are missing?
and build #3296 from today also doesn't show any changes.

**Reply by OBattler** [reply]
I belive build #3296 has no changes as it was a manual build started by RichardG.

**Reply by honx4amiga** [reply]
have there been no changes for latest 20-30 builds (#3573, #3574, #3575, #3576, #3577, #3578, #3579, #3580, #3581, #3582, #3583, #3584, #3585, #3586, #3587, #3588, #3589, #3590, #3591, #3592, #3593, #3594, #3595, #3596, #3597, #3598, #3599, #3600)?
at least none shown on https://ci.86box.net/job/86Box/changes page. right after build #3572 build #3601 is listed.

**Reply by dhrdlicka** [reply]
Build numbers can be skipped on the team's discretion.
